R/colourInput.R

```diff
@@ -194,59 +205,19 @@ colourInput <- function(inputId, label, value = "white",
 #' @export
 updateColourInput <- function(session, inputId, label = NULL, value = NULL,
                               showColour = NULL, palette = NULL, allowedCols = NULL,
-                              allowTransparent = NULL, transparentText = NULL,
+                              allowAlpha = NULL, allowTransparent = NULL, transparentText = NULL,
                               returnName = NULL) {
   message <- dropNulls(list(
-    label = label, value = formatHEX(value),
```

Comment:
(but also, it makes me a bit uneasy that this commit both adds alpha AND adds ability to type in colour names AND removes this hex format dependency AND whatever else amazing stuff you did in the plugin :)